code/Common/src/main/java/net/selenate/common/SelenateUtils.java

```diff
@@ -1,17 +1,39 @@
 package net.selenate.common;
 
 import java.util.Optional;
-
 import java.text.SimpleDateFormat;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Set;
 
+import net.selenate.common.exceptions.SeEmptyArgumentListException;
+import net.selenate.common.exceptions.SeNullArgumentException;
+
 public final class SelenateUtils {
   private SelenateUtils() { }
 
   public static final SimpleDateFormat ISO_8601_FORMAT = new SimpleDateFormat("yyyy-MM-dd HHmmss");
 
+  // ---------------------------------------------------------------------------
+  // G U A R D Z
+
+  public static <T> T guardNull(final T t, final String field) {
+    if (t == null) throw new SeNullArgumentException(field);
+    return t;
+  }
+
+  public static <T> List<T> guardEmptyList(final List<T> tList, final String field) {
```

Comment:
Throw exception on null element. There is no excuse for having null elements in lists.